pkg/containers/containers.go

```diff
@@ -41,25 +41,31 @@ type CgroupInfo struct {
 
 // InitContainers initializes a Containers object and returns a pointer to it.
 // User should further call "Populate" and iterate with Containers data.
-func InitContainers() *Containers {
+func InitContainers() (*Containers, error) {
 	cgroupV1 := false
 	if _, err := os.Stat("/sys/fs/cgroup/cgroup.controllers"); os.IsNotExist(err) {
 		cgroupV1 = true
-		initCgroupV1()
+		if err := initCgroupV1(); err != nil {
+			return nil, nil
```

Comment:
Shouldn't this return an error? If not perhaps a debug statement?

Reply:
Yes, you are right. Good catch